<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aplikacj Miasta Trasa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="labelDistance">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Długość trasy:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="buttonFindPath">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #e0e0e0;
    border: 1px solid #a0a0a0;
    padding: 5px 10px;
    font-size: 17px;
}

QPushButton:hover {
    background-color: #d0d0d0;
}
</string>
      </property>
      <property name="text">
       <string>Znajdź trasę</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTextBrowser" name="textBrowserLength">
      <property name="styleSheet">
       <string notr="true">QTextBrowser {
    background-color: #f5f5f5;
    border: 1px solid #b0b0b0;
    padding: 6px;
    font-size: 13px;
}
</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextBrowser" name="textBrowserPath">
      <property name="styleSheet">
       <string notr="true">QTextBrowser {
    background-color: #f5f5f5;
    border: 1px solid #b0b0b0;
    padding: 6px;
    font-size: 13px;
}
</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="5">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBoxStart">
      <property name="font">
       <font>
        <family>Titillium Web</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    background-color: #e0e0e0;     
    border: 1px solid #a0a0a0;   
    padding: 4px;
    font-size: 13px;
}
</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelStart">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Miasto początkowe:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBoxEnd">
      <property name="font">
       <font>
        <family>Titillium Web</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    background-color: #e0e0e0;     
    border: 1px solid #a0a0a0;   
    padding: 4px;
    font-size: 13px;
}
</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelEnd">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Miasto końcowe:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelPath">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Najkrótsza trasa:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Autor: Aleksandra Konopelska</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1364</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
